### PR TITLE
Implement MOTD command

### PR DESCRIPTION
**Pull request by marmichalski**

### CODE REVIEW

File: src/server.rs [hunk 1]
```diff
@@ -3,6 +3,9 @@ use crate::client::Client;
 use crate::replies::NumericReply;
 
 use std::collections::HashMap;
+use std::fs::File;
+use std::io::BufRead;
+use std::io::BufReader;
```
Comment (diath) [inline src/server.rs]: This can be replaced with:
```suggestion
use std::io::{BufRead, BufReader};
```

File: src/server.rs [hunk 2]
```diff
@@ -33,9 +37,25 @@ impl Server {
...
+            return None;
+        }
+
+        let mut lines = vec![];
+        let reader = BufReader::new(file.unwrap());
+        for line in reader.lines() {
+            lines.push(line.unwrap());
+        }
+
+        return Some(lines);
```
Comment (diath) [inline src/server.rs]: This could be just:
```suggestion
        Some(lines)
```

**Review by diath — CHANGES_REQUESTED** [review]
The `send_motd` function should also be called in `Client::complete_registration` at the end, and could also get word wrapped around 100 characters like the rest of the codebase.

File: src/server.rs (hunk 1 above)
Comment (marmichalski) [reply src/server.rs]: 🤢

**Pull request merged**